CMakeLists.txt

```diff
@@ -16,11 +22,66 @@ if(ALPAKA_ACC_GPU_CUDA_ENABLE)
     set(CUDA_NVCC_FLAGS "${CUDA_NVCC_FLAGS};--expt-extended-lambda")
 endif()
 
-# Use doxygen for api doc when available.
-find_package(Doxygen QUIET)
-if(DOXYGEN_FOUND)
+
+add_library(${PROJECT_NAME} INTERFACE)
+target_include_directories(${PROJECT_NAME}
+  INTERFACE
+  $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/include>
+  $<INSTALL_INTERFACE:include>)
+target_link_libraries(${PROJECT_NAME}
+  INTERFACE
+  alpaka::alpaka)
+add_library(vikunja::vikunja ALIAS ${PROJECT_NAME})
+
+
+if(${CMAKE_PROJECT_NAME} STREQUAL ${PROJECT_NAME})
```

Comment:
So this is checking if vikunja is used as a subdirectory?

Reply:
yes